setup.py

```diff
@@ -16,7 +16,7 @@ def package_data(pkg, root):
 
 setup(
     name='recommender-xblock',
-    version='0.1',
+    version='0.2',
```

Comment:
@mattdrayer Agreed. The two values should be the same.

Of the two, `setup.py` is outdated. The code reached RC status after we had robust data of learning improvements. I think we hit 1.0 with the addition of i18n.